samples/observability-traces-otlp.js

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright 2025 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//   title: Observability (Tracing) with OpenTelemetry using OTLP
+//   usage: node observability-traces-otlp.js <PROJECT-ID> <INSTANCE-ID> <DATABASE-ID>
+
+'use strict';
+
+async function main(
+  projectId = 'my-project-id',
+  instanceId = 'my-instance-id',
+  databaseId = 'my-project-id'
+) {
+  // [START spanner_opentelemetry_traces_otlp_usage]
+  const {NodeTracerProvider} = require('@opentelemetry/sdk-trace-node');
+  const {
+    OTLPTraceExporter,
+  } = require('@opentelemetry/exporter-trace-otlp-grpc');
+  const {BatchSpanProcessor} = require('@opentelemetry/sdk-trace-base');
+  const {TraceIdRatioBasedSampler} = require('@opentelemetry/sdk-trace-base');
+  const {Resource} = require('@opentelemetry/resources');
+  const {Spanner} = require('@google-cloud/spanner');
+
+  // Define a Resource with service metadata
+  const resource = new Resource({
+    'service.name': 'my-service',
+    'service.version': '1.0.0',
+  });
+
+  // Create an OTLP gRPC trace exporter
+  const traceExporter = new OTLPTraceExporter({
+    url: 'http://localhost:4317', // Default OTLP gRPC endpoint
+  });
+
+  // Create a provider with a custom sampler
+  const provider = new NodeTracerProvider({
+    sampler: new TraceIdRatioBasedSampler(1.0), // Sample 100% of traces
+    resource,
+    spanProcessors: [new BatchSpanProcessor(traceExporter)],
+  });
+
+  // Uncomment following line to register tracerProvider globally
```

Comment:
```suggestion
  // Uncomment following line to register tracerProvider globally instead of passing it into SpannerOptions.observabilityOptions.
```

can we update the comment as above to let the customers know that they don't need to pass the tracerProvider in the ObservabilityOptions if they are registering tracerProvider globally ?